### DISCUSSION TITLE
Navigation Tabs with multiple Languages

### DISCUSSION THREAD

**Discussion by dnylive** [squidfunk/mkdocs-material#2228] [Q&A] [answered]
Hi there,

first and foremost, thanks for your awesome work.
I can't get the Navigation tabs to work with multiple languages.

How does the mkdocs.yml "nav:" part have to look to work?

Thanks

**Comment by squidfunk** [comment, marked as answer]
I assume you mean the [language selector](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language-selector), which is effectively a dropdown with links that should point to separate MkDocs projects. For example, if you want to support `en`, `fr` and `de`, it is recommended that you create the following file structure:

```
.
├─ en/
│  ├─ docs/
│  └─ mkdocs.yml
├─ fr/
│  ├─ docs/
│  └─ mkdocs.yml
└─ de/
   ├─ docs/
   └─ mkdocs.yml
```

Then, in each of the `mkdocs.yml`, add the language selector configuration [as explained in the documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language-selector). Note that in each `mkdocs.yml`, you should have defined `site_url`with the respective language, i.e. `<your-site>/en/`. The navigation itself is self-contained for each language.

**Reply by dnylive** [reply]
Thanks, that worked perfectly.

**Reply by sleepwalker2017** [reply]
Hi I'd like to ask how to deploy this project. Seems mkdocs serve will not work.

**Reply by squidfunk** [reply]
You can serve and build each instance separately, but you can't use `mkdocs gh-deploy` to deploy the site. You would need a separate deployment script. However, as I understand the [mkdocs-static-i18n](https://github.com/ultrabug/mkdocs-static-i18n) plugin solves this problem.
